### PR TITLE
F save targets

### PR DESCRIPTION
**Pull request by dg-shadow**

### CODE REVIEW

File: sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py [hunk 1]
```diff
@@ -10,22 +10,45 @@
...
 
         self.__name = name
-
         self.__hand_h = hand_h
-
-        if hand_or_arm == "arm":
-            self.__commander = SrArmCommander()
-
-        elif hand_or_arm == 'hand':
+        self.__save_target = save_target
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py]: I hate these double leading underscores. We should always use single underscores.

Comment (dg-shadow) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py]: they have a purpose, but ok ;p

Comment (dg-shadow) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py]: As in, double underscore is technically more "correct" but happy to use the dirty, easier to type, single underscore if you insist

Comment (dg-shadow) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py]: Although either way, changing to all single _ would make the diff quite messy. maybe merge this one and I'll do a separate pr to change the underscores?

File: sr_example/scripts/sr_example/hand_examples/sr_hand_optoforce_tactile_example.py [hunk 2]
```diff
@@ -1,24 +0,0 @@
-#!/usr/bin/env python
```
Comment (toliver) [inline sr_example/scripts/sr_example/hand_examples/sr_hand_optoforce_tactile_example.py]: Wow. Why does this appear as deleted?

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py (hunk 1 above)
Comment (toliver) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py]: Yes, indeed that would be a separate PR. And I mean it mainly for new files.

About it being more correct. that's far from true.

File: sr_example/scripts/sr_example/hand_examples/sr_hand_optoforce_tactile_example.py (hunk 2 above)
Comment (dg-shadow) [reply sr_example/scripts/sr_example/hand_examples/sr_hand_optoforce_tactile_example.py]: No idea o.O

**Comment by beatrizleon** [comment]
Closing this pr as #257 is now opened for kinetic

**Pull request closed without merging**